resolwe_bio/processes/import_data/seq_reads.yml

```diff
@@ -218,3 +218,248 @@
       re-save-file-list fastqc_archive2 "${RW_FASTQC[@]}"
       re-save-list fastqc_url "${FW_FASTQC_URL[@]}"
       re-save-list fastqc_url2 "${RW_FASTQC_URL[@]}"
+
+- slug: basespace-fastq-single
+  name: BaseSpace Reads (single-end)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/legacy:latest
```

Comment:
I am strongly against running any kind of bioinformatics tools (like fastqc) in processes which have `requirements.resources.secrets = true`. These processes should be using minimal Docker images, which can be more easily audited, without any of these tools. Any post-processing should be handled in another process and the combined flow should be encapsulated in a workflow.

I know that there are currently some missing features with workflows which make this inconvenient, but at least we should put up a big warning sign that this is currently less secure and a fixme that this must be fixed in order to improve security.

cc @tjanez @dblenkus

Reply:
It seems `resolwe/base` Docker image does not have `requests` installed.

Reply:
@kostko Should we extend the base image or create a new one for this process? @hadalin [This](https://github.com/genialis/resolwe-bio/tree/master/resolwe_bio/docker_images) is where we add Docker images with bioinformatics tools.

Reply:
@dblenkus recommended `resolwebio/utils` and it works.

Reply:
FYI `resolwebio/utils` has Python `2.7.14` installed.